.vsts-dotnet-ci.yml

```diff
@@ -13,71 +13,69 @@ resources:
 phases:
 - template: /build/ci/phase-template.yml
   parameters:
-    name: core30
-    buildScript: build.cmd
+    name: Centos_x64_NetCoreApp30
+    buildScript: ./build.sh
     customMatrixes:
-      Build_Debug_Intrinsics:
+      Debug_Build:
         _configuration: Debug-Intrinsics
         _config_short: DI
         _includeBenchmarkData: false
-      Build_Release_Intrinsics:
+      Release_Build:
         _configuration: Release-Intrinsics
-        _config_short: RI
+        _config_short: R
         _includeBenchmarkData: true
     queue:
-      name: Hosted VS2017
+      name: Hosted Ubuntu 1604
+      container: CentosContainer
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: Centos
+    name: Ubuntu_x64_NetCoreApp21
     buildScript: ./build.sh
-    customMatrixes:
-      Build_Debug_Intrinsics:
-        _configuration: Debug-Intrinsics
-        _config_short: DI
-        _includeBenchmarkData: false
-      Build_Release:
-        _configuration: Release
-        _config_short: R
-        _includeBenchmarkData: true
     queue:
       name: Hosted Ubuntu 1604
-      container: CentosContainer
+      container: UbuntuContainer
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: Ubuntu
+    name: MacOS_x64_NetCoreApp21
     buildScript: ./build.sh
+    queue:
+      name: Hosted macOS
+
+- template: /build/ci/phase-template.yml
+  parameters:
+    name: Windows_x64_NetCoreApp30
+    buildScript: build.cmd
     customMatrixes:
-      Build_Debug:
-        _configuration: Debug
-        _config_short: D
+      Debug_Build:
+        _configuration: Debug-Intrinsics
+        _config_short: DI
         _includeBenchmarkData: false
-      Build_Release_Intrinsics:
+      Release_Build:
         _configuration: Release-Intrinsics
         _config_short: RI
         _includeBenchmarkData: true
     queue:
-      name: Hosted Ubuntu 1604
-      container: UbuntuContainer
+      name: Hosted VS2017
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: MacOS
-    buildScript: ./build.sh
+    name: Windows_x64_NetCoreApp21
+    buildScript: build.cmd
     queue:
-      name: Hosted macOS
+      name: Hosted VS2017
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: Windows_NetFx
+    name: Windows_x64_NetFx461
     buildScript: build.cmd
```

Comment:
explicit `architecture: x64` ?

Reply:
Actually the default value of this architecture variable is x64 which we set here https://github.com/dotnet/machinelearning/pull/3553/files/251954988b8ebd8e2dd8c1eb420a5e7c09095dac#diff-51ce39ab4212d73da272ba04e66b0b3bR3